lib/generators/apple_auth/apple_auth_controller/templates/apple_auth_controller.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class <%= @scope_prefix %>AppleSignInController < DeviseTokenAuth::SessionsController
+class <%= @scope_prefix %>AppleAuthController < DeviseTokenAuth::SessionsController
```

Comment:
Maybe (later) we can update the [rails repo](https://github.com/rootstrap/apple-sign-in-rails) to reflect this changes